elasticsearch_loader/iter.py

```diff
@@ -1,5 +1,5 @@
 try:
-    from itertools import izip_longest as zip_longest
+    from itertools import zip_longest as zip_longest
 except ImportError:
     from itertools import zip_longest
```

Comment:
no need for the entire `try catch` block, `from itertools import zip_longest as zip_longest` is enough